src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
 // write code here
+'use strict';
```

Comment:
Don't need to duplicate 'use strict' 
<img width="322" alt="image" src="https://github.com/mate-academy/js_task_parse_list_DOM/assets/64750958/e51400d1-a3f4-453c-b79c-5aeb1c1cb8bc">
```suggestion
'use strict';
```